Google.PowerShell/Sql/GcSqlInstanceCmdlets.cs

```diff
@@ -418,15 +422,20 @@ protected override void ProcessRecord()
     /// <summary>
     /// <para type="synopsis">
     /// Imports data into a Cloud SQL instance from a MySQL dump 
-    /// or CSV file in a Google Cloud Storage bucket. 
+    /// or CSV file stored either in a Google Cloud Storage bucket or on your local machine.
     /// </para>
     /// <para type="description">
     /// Imports data into a Cloud SQL instance from a MySQL dump 
-    /// or CSV file stored in a Google Cloud Storage bucket.
+    /// or CSV file stored either in a Google Cloud Storage bucket or on your local machine.
     /// 
     /// Only one database may be imported from a MySQL file,
     /// and only one table may be imported from a CSV file.
     /// </para>
+    /// <para>
+    /// WARNING: Standard charging rates apply if a file is imported from your local machine.
+    /// A Google Cloud Storage bucket will be set up, uploaded to, imported from,
+    /// and deleted during the local upload process.
```

Comment:
It will not be deleted _during_ the upload process. It will be deleted "after the upload and import process completes".